doc/ref_arch/kubernetes/chapters/chapter02.rst

```diff
@@ -4,9 +4,9 @@ Architecture Requirements
 Introduction
 ------------
 
-This chapter will use the requirements defined in the overall Reference Model and only make additional entries in
-section `Kubernetes Architecture Requirements <#kubernetes-architecture-requirements>`__ if there are additional
-requirements needed for this Reference Architecture.
+This chapter will translate the requirements defined in the overall Reference Model into Kubernetes-specific
```

Comment:
IMHO, the Reference Model (RM) specifies a set of generic technology/implementation agnostic requirements and the process of deriving the Reference Architecture requirements from the RM requirements is one of specializing rather than one of translation.

Reply:
thanks, fixed in https://github.com/cntt-n/CNTT/pull/3276/commits/1f65f8751e97634b2a1f178d4198aee40f8c3529